# which table to face
int32 table_index
---
bool success
